src/flint/__init__.py

```diff
@@ -21,5 +21,5 @@
 from .types.acb_mat import *
 from .types.acb_series import *
 from .types.fmpz_mpoly import *
-
+from .functions.showgood import showgood
 __version__ = '0.4.4'
```

Comment:
I suggest putting two blank lines between these. The cause of the merge conflict is basically just not having a blank line here.